tests/test_events.py

```diff
@@ -17,52 +17,108 @@
+                # BaseLogEntry
+                "level": "info",
+                "text": "some log message\u0020line 2",
+                "timestamp": "__any_value__",
+                "stackTrace": {
+                    "callFrames": [{
+                        "url": "",
+                        "functionName": "",
+                        "lineNumber": 1,    # TODO: lineNumber should start from 0
+                        "columnNumber": 8}]},
+                # ConsoleLogEntry
+                "type": "console",
+                "method": "log",
+                "realm": context_id,
+                "args": [{
+                    "type": "string",
+                    "value": "some log message"}, {
+                    "type": "string",
+                    "value": "line 2"}]}},
+        log_entry_added_event_response,
+        ["timestamp"])
 
     # Assert command done.
-    resp = await read_JSON_message(websocket)
-    assert resp == {
+    assert command_done_response == {
         "id": 33,
         "result": {"type": "undefined"}}
 
+    # Send command.
+    await send_JSON_command(websocket, {
+        "id": 34,
+        "method": "script.evaluate",
+        "params": {
+            "expression": "console.log('format specificier, string: %s, integer: %d, integer: %i, float: %f','line 2', 1, -2, 1.234)",
+            "target": {"context": context_id}}})
+
+    # Wait for responses
+    log_entry_added_event_response, command_done_response = \
+        await wait_for_event(websocket)
+
+    # Assert "log.entryAdded" event emitted.
+    recursiveCompare(
+        {
+            "method": "log.entryAdded",
+            "params": {
+                # BaseLogEntry
+                "level": "info",
+                "text": "format specificier, string: line 2, integer: 1, integer: -2, float: 1.234",
+                "timestamp": "__any_value__",
+                "stackTrace": {
+                    "callFrames": [{
+                        "url": "",
+                        "functionName": "",
+                        "lineNumber": 1,    # TODO: lineNumber should start from 0
+                        "columnNumber": 8}]},
+                # ConsoleLogEntry
+                "type": "console",
+                "method": "log",
+                "realm": context_id,
+                "args": [{
+                    "type": "string",
+                    "value": "format specificier, string: %s, integer: %d, integer: %i, float: %f"}, {
+                    "type": "string",
+                    "value": "line 2"},{
+                    "type": "number",
+                    "value": 1}, {
+                    "type": "number",
+                    "value": -2}, {
+                    "type": "number",
+                    "value": 1.234}]}},
+        log_entry_added_event_response,
+        ["timestamp"])
+
+    # Assert command done.
+    assert command_done_response == {
+        "id": 34,
+        "result": {"type": "undefined"}}
```

Comment:
Can be emitted, as tested in other place, and out of scope for the given test.

Reply:
emitted as suggested.